app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,20 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(
+            self, nickname: str,
+            musical_instrument: str,
+            bow_level: int
+    ) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def player_info(self) -> None:
+        return (
+            f"Elf ranger {self.nickname}. "
+            f"{self.nickname} has bow of the {self._bow_level} level"
+        )
+
+    def get_rating(self) -> int:
+        return 3 * self._bow_level
```

Comment:
Code Style: It's a good practice to use type hinting for your methods and functions. Please add type hinting to the `get_rating` method. It should look like this: `def get_rating(self) -> int:`.